.github/workflows/support.yml

```diff
@@ -0,0 +1,33 @@
+# Configuration for support-requests - https://github.com/dessant/support-requests
+name: 'Support Requests'
+
+on:
+  issues:
+    types: [labeled, unlabeled, reopened]
+
+permissions:
+  issues: write
+
+jobs:
+  mark-duplicate:
```

Comment:
looks like c&p from another workflow

Reply:
Ooops, let me remove it and test out that it works.

Reply:
I've changed it to `mark-support`.